wrappers/golang/curves/bw6761/g2_base_field.go

```diff
@@ -0,0 +1,81 @@
+//go:build g2
```

Comment:
Technically g2 base field for this curve is exactly the same as g1 base field but if it's more convenient to define and test it separately for some template-related reason, I don't think it's a big issue.